src/Common/Constants.ts

```diff
@@ -362,6 +365,13 @@ export class Notebook {
     "We have identified an issue with the Mongo Shell and it is unavailable right now. We are actively working on the mitigation.";
   public static readonly cassandraShellTemporarilyDownMsg =
     "We have identified an issue with the Cassandra Shell and it is unavailable right now. We are actively working on the mitigation.";
+  public static saveNotebookModalTitle = "Save Notebook in temporary environment";
+  public static saveNotebookModalContent =
+    "Notebook Will be saved in temporary environment that lasts for a dedicated period of time. To Store and run notebook for a longer period, please connect to your Github repo and move your notebooks to Github instead.";
+  public static newNotebookModalTitle = "Create Notebook in temporary environment";
+  public static newNotebookUploadModalTitle = "Upload Notebook in temporary environment";
+  public static newNotebookModalContent =
+    "A temporary environment will be created everytime you need to create, view, edit, and run a notebook for a dedicated period of time. To Store and run notebook for a longer period, please connect to your Github repo and move your notebooks to Github instead.";
```

Comment:
Suggested text: 

A temporary workspace will be created to enable you to work with notebooks. When the session expires, any notebooks in the workspace will be removed. 

To save your work permanently, save your notebooks to a GitHub repository or download the notebooks to your local machine before the session ends. [Learn more.](https://aka.ms/cosmos-notebooks-limits)